Package.swift

```diff
@@ -24,6 +24,11 @@
 
 import PackageDescription
 
+// To use a local installation of ArcGIS Runtime for iOS, uncomment the following lines and also uncomment
+// package depenencies:package:path and comment the depenencies:package:url
+// import class Foundation.ProcessInfo
+// let sdk_install_package: String = (ProcessInfo.processInfo.environment["HOME"] ?? "") + "/Library/SDKs/ArcGIS"
```

Comment:
```suggestion
// let localArcGISPackage = ProcessInfo.processInfo.environment["HOME"]! + "/Library/SDKs/ArcGIS"
```

snake case isn't common in swift. And then I think that `ProcessInfo.processInfo.environment["HOME"]!` should be safe to force unwrap. Also, we don't include the type when the compiler can infer it.